src/main/java/no/digipost/signature/client/core/internal/SignerCustomizations.java

```diff
@@ -31,4 +32,6 @@
      */
     B withSignatureType(SignatureType type);
 
+    B withOnBehalfOf(OnBehalfOf onBehalfOf);
```

Comment:
Hva med bare `onBehalfOf(…)`? `signOnBehalfOf(…)`? `signingOnBehalfOf(…)`?
Synes de kanskje leser litt bedre enn `withOnBehalfOf(…)`

Du kan vurdere om det er verdt å gå bort fra standard-mønsteret på de metodenavnene her for marginalt bedre lesbarhet 💃